src/main/java/seedu/application/logic/commands/DeleteInterviewCommand.java

```diff
@@ -45,7 +44,8 @@ public CommandResult execute(Model model) throws CommandException {
         Application editedApplication = new Application(applicationToRemoveInterview);
         model.setApplication(applicationToRemoveInterview, editedApplication);
         model.updateApplicationListWithInterview();
-        model.updateFilteredApplicationList(PREDICATE_SHOW_ALL_APPLICATIONS);
+
+        assert applicationToRemoveInterview.getInterview().isEmpty();
```

Comment:
```suggestion
        assert applicationToRemoveInterview.getInterview().isPresent();
```
Is this more logical? Though anyway for some reason it doesn't seem to matter if `isEmpty` or `isPresent` is used here when I changed and tested. Seems a little odd since logically only one should be true?

Reply:
I used isPresent initially also, but the IDE suggested to change to isEmpty haha. Should be same result for both so I think it doesn't matter. 😂